README.md

```diff
@@ -1,3 +1,11 @@
+# This repository has moved!
+
+It now lives in the repository of the [SonarJS analyzer](https://github.com/SonarSource/SonarJS/blob/master/packages/jsts/src/rules/README.md).
+
+eslint-plugin-sonarjs >2.0.0 makes all SonarJS rules available for ESLint users, instead of a subset as it was with ^1.0.0 living here.
```

Comment:
```suggestion
eslint-plugin-sonarjs >= 2.0.0 makes all SonarJS rules available for ESLint users, instead of a subset as it was with ^1.0.0 living here.
```